backend/src/repositories/teamCreationRepository.js

```
@@ -0,0 +1,44 @@
+const getAssociatedDisaster = (organization, disasterId) => {
+    if (disasterId) {
+      const disaster = organization.disasters.find(
+        d => d.disaster_id === parseInt(disasterId, 10) && d.status !== 'Closed'
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)